lib/src/controller/chat_controller.dart

```diff
@@ -41,7 +42,7 @@ class ChatController {
   ///    chatcontroller.typingIndicatorNotifier.addListener((){});
   ///  ```
   /// For more functionalities see [ValueNotifier].
-  ValueNotifier<bool> get typingIndicatorNotifier => _showTypingIndicator;
+  ValueListenable<bool> get typingIndicatorNotifier => _showTypingIndicator;
```

Comment:
I don't see any specific reason for this change. Can you please revert this or let me know if I am missing anything?

Reply:
`ValueListenable` does not have a setter.